Unable to start web service after `pnpm build` with suspicous pnpm lockfile

### Self Checks

- [x] I have read the [Contributing Guide](https://github.com/langgenius/dify/blob/main/CONTRIBUTING.md) and [Language Policy](https://github.com/langgenius/dify/issues/1542).
- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [x] I confirm that I am using English to submit this report, otherwise it will be closed.
- [x] 【中文用户 & Non English User】请使用英语提交，否则会被关闭 ：）
- [x] Please do not modify this template :) and fill in all the required fields.

### Dify version

main

### Cloud or Self Hosted

Self Hosted (Source)

### Steps to reproduce

- Put a `pnpm-lock.yaml` file into users's home folder, which is occasionally added when installing openclaw or else via pnpm package manager
- Failed to start web service to build and run with 
```
pnpm build 
pnpm start
```
comes with the following errors of "Cannot find module '/Users/bw/dev/dify/web/.next/standalone/server.js'":

```
(dify-api) bw@bwmac2024 web % pnpm start

> dify-web@1.13.2 start /Users/bw/dev/dify/web
> node ./scripts/copy-and-start.mjs

Starting copy-and-start script
Starting directory copy operations
Ensuring destination parent directory exists: .next/standalone/.next
Checking if path exists: .next/static
Path exists: .next/static
Copying directory from .next/static to .next/standalone/.next/static
Successfully copied .next/static to .next/standalone/.next/static
Ensuring destination parent directory exists: .next/standalone
Checking if path exists: public
Path exists: public
Copying directory from public to .next/standalone/public
Successfully copied public to .next/standalone/public
Finished directory copy operations
Starting server on 0.0.0.0:3000
Server script path: .next/standalone/server.js
Environment variables - PORT: 3000, HOSTNAME: 0.0.0.0
node:internal/modules/cjs/loader:1459
  throw err;
  ^

Error: Cannot find module '/Users/bw/dev/dify/web/.next/standalone/server.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1456:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1066:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1071:22)
    at Module._load (node:internal/modules/cjs/loader:1242:25)
    at wrapModuleLoad (node:internal/modules/cjs/loader:255:19)
    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:154:5)
    at node:internal/main/run_main_module:33:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v24.14.0
Server exited with code: 1
 ELIFECYCLE  Command failed with exit code 1.
```

### ✔️ Expected Behavior

- successfully start the web service

### ❌ Actual Behavior

_No response_

--- a/web/next.config.ts
+++ b/web/next.config.ts
@@ -10,6 +10,7 @@ const nextConfig: NextConfig = {
   basePath: env.NEXT_PUBLIC_BASE_PATH,
   transpilePackages: ['@t3-oss/env-core', '@t3-oss/env-nextjs', 'echarts', 'zrender'],
   turbopack: {
+    root: process.cwd(),
     rules: codeInspectorPlugin({
       bundler: 'turbopack',
     }),

